lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -1261,11 +1260,24 @@ class SemanticsObject {
   bool get isTextField => hasFlag(ui.SemanticsFlag.isTextField);
 
   /// Whether this object represents a heading element.
+  ///
+  /// Typically, a heading is a prominent piece of text that describes what the
+  /// rest of the screen or page is about.
+  ///
+  /// Not to be confused with [isHeader].
   bool get isHeading => headingLevel != 0;
 
-    /// Whether this object represents an editable text field.
+  /// Whether this object represents an editable text field.
```

Comment:
```suggestion
  /// Whether this object represents an interactive link.
```

This doc seems wrong? This isn't talking about `ui.SemanticsFlag.isTextField`